src/migration/ember-addon/steps/augment-options.js

```diff
@@ -68,8 +68,14 @@ function analyzePackageManager(options) {
 
 function deriveAddonLocation(addonPackage) {
   if (!addonPackage.name) {
-    throw new RangeError(
-      `ERROR: In package.json, the package name \`${addonPackage.name}\` is not valid.`
+    throw new SyntaxError(
+      'ERROR: In package.json, the package name is missing.'
+    );
+  }
+
+  if (!addonPackage.version) {
+    throw new SyntaxError(
+      'ERROR: In package.json, the package version is missing.'
```

Comment:
According to `npm`, we can expect that a valid `package.json` will always have the `name` and `version` fields.

https://docs.npmjs.com/creating-a-package-json-file#required-name-and-version-fields